server/swap/swap.go

```diff
@@ -44,10 +44,10 @@ func unixMsNow() time.Time {
 type AuthManager interface {
 	Route(string, func(account.AccountID, *msgjson.Message) *msgjson.Error)
 	Auth(user account.AccountID, msg, sig []byte) error
-	Sign(...msgjson.Signable)
+	Sign(...msgjson.Signable) error
 	Send(account.AccountID, *msgjson.Message)
 	Request(account.AccountID, *msgjson.Message, func(comms.Link, *msgjson.Message))
-	Penalize(account.AccountID, *order.Match, order.MatchStatus)
```

Comment:
The reason for providing order.Match and order.MatchStatus isn't clear, but the implementation in the `auth` package just requires the `account.Rule`.

Reply:
It's not clear to me what will be needed here either. Good for now.